genai-perf/genai_perf/parser.py

```diff
@@ -652,6 +684,23 @@ def _add_other_args(parser):
     )
 
 
+def _add_goodput_args(parser):
+    goodput_group = parser.add_argument_group("Goodput")
```

Comment:
We shouldn't be adding a whole group just for this specific feature. I would put this under the input group.